src/clr-core/.storybook/main.js

```diff
@@ -26,13 +25,15 @@ module.exports = {
     });
 
     config.resolve.extensions.push('.ts');
-    config.resolve.extensions.push('.js');
-    config.resolve.alias = {
-      '@clr/core': path.resolve(__dirname, '../../../dist/clr-core'),
-      '@clr/core/*': path.resolve(__dirname, '../../../dist/clr-core/*'),
-    };
```

Comment:
We dont need storybook to resolve these files as TS does this already with its path mapping